### ISSUE TITLE
[Every version] Hand tools become unwieldable if  i install Extender 

### ISSUE THREAD

**Issue by Luqin12** [SEModCommunity/SE-Community-Mod-API#163]
So far everyone has this issue who tried to install/update Extender. Everything works.. but all clients cannot wield hand tools.. if they try to push the button, the tool just flickers on screen and unwields itself immediatelly.

We all who tried it have one thing in common.. we are from the Czech Republic.. we have a suspicion that this might be related. In our langugae welder is called "svářeč" and grinder is called "drtič" and drill is called "vrtačka". 

Each of them has pretty alien letters from english perspective...

Please help us with this, Space Engineers is a Czech game it would be sad if we cannot play  :-( 

(we tested it on 4 computers very far from each other.. all same problem)

**Comment by Moustachauve** [comment]
Can you test if it still happen in lastest dev release please?

**Comment by Luqin12** [comment]
Yes, the problem is still there. And before this version we tested every ServerExtender release down.. because we wanted newest possible. But none worked.

**Comment by Moustachauve** [comment]
It is normal that older version don't work.

Can you post the log from a game where it doesn't work in the lastest dev build ( https://github.com/SEModCommunity/SE-Community-Mod-API/releases/tag/v0.2.10.12-extender-alpha )?

**Comment by AustinCZ** [comment]
Hi,
latest version 0.2.10.13
console log:
Finished loading LogManager
Finished loading SandboxGameAssemblyWrapper
All main types passed reflection unit tests!
All entity types passed reflection unit tests!
All block types passed reflection unit tests!
Finished creating server!
Finished loading ServerAssemblyWrapper
Finished loading PluginManager
Finished loading FactionsManager
Finished loading EntityEventManager
Finished loading ChatManager
Auto-Start enabled
Common instance pre-selected: 'c:\hlserver\27556\server'
GUI disabled
SLWCF disabled
WCF port: 26556
Space engineers 01_048_021
Is official: True [O]
Environment.Is64BitProcess: True
2014-09-23 15:30:12.744: Bind IP : 0.0.0.0:27556
2014-09-23 15:30:13.886: Server successfully started
2014-09-23 15:30:13.886: Product name: Space Engineers
2014-09-23 15:30:13.886: Desc: Space Engineers
2014-09-23 15:30:13.886: Public IP: 93.91.250.107
2014-09-23 15:30:13.886: Steam ID: 90092060564841479
2014-09-23 15:30:15.436: Loading session: C:\hlserver\27556\server\Saves\Created 2014-08-18 0145
2014-09-23 15:31:08.042: Session loaded
2014-09-23 15:31:08.090: Server connected to Steam
2014-09-23 15:31:08.092: Server PolicyResponse (1)
Loading plugins ...
Finished loading plugins
Initializing plugins ...
Finished initializing plugins
2014-09-23 15:31:09.124: Game ready... Press Ctrl+C to exit
Finished loading NetworkManager
Finished loading PlayerManager
Finished loading WorldManager
2014-09-23 15:35:12.293 - Save complete and took 1.2330705 seconds
2014-09-23 15:36:26.801: Peer2Peer_SessionRequest 76561197961765570
2014-09-23 15:36:31.699: OnConnectedPlayer Austin attempt
2014-09-23 15:36:32.052: Server ValidateAuthTicketResponse (k_EAuthSessionResponseOK), owner: 76561197961765570
2014-09-23 15:36:32.432: World request received: Austin
2014-09-23 15:40:11.674 - Save complete and took 0.6060347 seconds
2014-09-23 15:45:11.648 - Save complete and took 0.5630322 seconds

---

2014-09-23 15:54:41.888 - Thread: 1 -> Log Started
2014-09-23 15:54:41.893 - Thread: 1 -> Timezone (local - UTC): 2h
2014-09-23 15:54:41.893 - Thread: 1 -> App Version: 0.2.3.10
2014-09-23 15:54:41.893 - Thread: 1 -> Starting SEServerExtender with 5 arguments ...

---

No errors anywhere.

**Comment by LordXaosa** [comment]
DedicatedServer64/Logs, there should be error log. Paste it here, pls.

**Comment by AustinCZ** [comment]
It is empty, no errors.

**Comment by Luqin12** [comment]
We have errors when we load game with the default extender on our hosted server... 
[FakaHeda] Successfully authorized 178.17.4.150!
Name-Value Arg: name='wcfport' value='26319'
Name-Value Arg: name='gamepath' value='c:\hlserver\27319\server'
Name-Value Arg: name='instance' value='c:\hlserver\27319\server'
Finished loading SandboxGameAssemblyWrapper
Failed to find method '7F81EFB4976A4245FF695F313211B544' in type '5F381EA9388E0A32A8C817841E192BE8.48D79F8E3C8922F14D85F6D98237314C'
BaseEntityNetworkManager reflection validation failed!
Failed to find method 'get_Integrity' in type '6DDCED906C852CFDABA0B56B84D0BD74.54A8BE425EAC4A11BFF922CFB5FF89D0'
CubeBlockEntity reflection validation failed!
Failed to find method 'D979DB9AA474782929587EC7DE5E53AA' in type '6DDCED906C852CFDABA0B56B84D0BD74.7085736D64DCC58ED5DCA05FFEEA9664'
FunctionalBlockEntity reflection validation failed!
Failed to find method '623B0AC0E7D9C30410680C76A55F0C6B' in type '33FB6E717989660631E6772B99F502AD.DE48496EE9812E665B802D5FE9E7AD77'
InventoryEntity reflection validation failed!
Failed to find method '6C26188EB12DAFB18299490EA313E2E3' in type 'FB8C11741B7126BD9C97FE76747E087F.7E69388ED0DB47818FB7AFF9F16C6EDA'
Failed to find method '7920DC5385A89CD682290EB437CC39D1' in type 'FB8C11741B7126BD9C97FE76747E087F.7E69388ED0DB47818FB7AFF9F16C6EDA'
Failed to find method '564ED006EA9A5F014460AF4DB324A9D1' in type 'FB8C11741B7126BD9C97FE76747E087F.7E69388ED0DB47818FB7AFF9F16C6EDA'
PowerProducer reflection validation failed!
Failed to find method '79BD2994D8EC029801BD5978D7474622' in type '5BCAC68007431E61367F5B2CF24E2D6F.8F510E70FE6A50C0B39D09689C2D6CF4'
GravityGeneratorEntity reflection validation failed!
Finished creating server!

**Comment by AustinCZ** [comment]
Those errors are from old version of SE, they are not related to this problem..

**Comment by Luqin12** [comment]
There is no more clarification needed... there are no errors to report. The extender just forbids tools on all czech localizations of windows.

**Comment by fvgoulet** [comment]
There is internationalization support to be done throughout the whole API, we need to investigate this issue further.

**Comment by Ragamuff** [comment]
Hi, I have this issue too.
There is no errors in console
Please try to put little bit more focus on this as its pathetic to czech users not beying able to use extender!

**Comment by Moustachauve** [comment]
This is the old "abandoned" git repo for this project. Please report this error on this repo: https://github.com/Tyrsis/SE-Community-Mod-API
